processing/src/main/java/io/druid/segment/SegmentDesc.java

```diff
@@ -22,8 +22,13 @@
 import com.google.common.base.Function;
 import com.metamx.common.logger.Logger;
 import io.druid.timeline.DataSegment;
+import org.apache.commons.lang.StringUtils;
 import org.joda.time.DateTime;
 import org.joda.time.Interval;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+
+import java.util.Objects;
 
 /**
  * identifier to DataSegment. wishfully included in DataSegment
```

Comment:
since DataSegment is moved to main druid repo, can you also move this to DataSegment class ?